spec/factories/registration.rb

```diff
@@ -14,6 +14,10 @@
 
     registration_exemptions { build_list(:registration_exemption, 3) }
 
+    trait :was_renewed do
+      referring_registration_id { true }
```

Comment:
😮This is interesting stuff. What is saved in the database? Probably a `1`. I would have expected Rails to complain about passing a Boolean to an integer field. 

Anyway, can we change this to: 
```suggestion
      referring_registration build(:registration)
```